examples/isomorphic/client/build-wasm.dev.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
Instead of .dev.sh maybe we can we rename all of these to .watch.sh since both are technically for development

Reply:
I stole `dev` name from `nuxt` :pizza: 
the unsuffixed versions are just the minimal script for quickly trying the project, not anything useful.
They avoid the extra cargo binary dependencies.

Reply:
Yeah, that is what it's doing after all.